### ISSUE TITLE
const declarations

### ISSUE THREAD

**Issue by gknocke** [Javex/libscep#75]
I think some configuration parameters should be declared as constant:
sigalg in scep_configuration for example:
currently, sigalg = EVP_get_digestbyname(md) raises a warning since its return type is declared as static. The same holds for encalg = EVP_get_cipherbyname(alg)

what do you think?

**Comment by Javex** [comment]
Yeah if it raises a warning, it'd probably not be such a bad idea. However, I recommend first getting up to speed on the correct handlng of const stuff. I mean we don't want to declare something const that we maybe change. For example, on the configuration, you actually _want_ to change it. The alternative would be casting the `const` away.
